ScriptSpecification.md

```diff
@@ -81,14 +81,16 @@ Everything after the number will be ignored for compilation and follows no speci
 | Type | Description |
 | - | - |
 | 30 | If the value equates to ``true``, jump to the label in the current function.<br>```if 1 goto "Label1"h;```<br>```if $variable0 goto "Label1"h;``` |
-| 31 | Unconditionally jump to the label in the current function.<br>```goto "Label1"h;``` |
-| 33 | If the value equates to ``false``, jump to the label in the current function.<br>```if 0 goto "Label1"h;```<br>```if $variable0 goto "Label1"h;``` |
+| 31 | Unconditionally jump to the label in the current function. Theoretically, if multiple labels are provided, it'll choose a random one - although this behaviour is unused and XtractQuery will require you to format it as `sub32(labelA, labelB...)` to use multiple. <br>```goto "Label1"h;```<br>```sub32("Label1"h, "Label2"h ...) // jumps to a random label``` |
+| 32 | If the value equates to ``true``, jump to the label in the current function.<br>```if 0 goto "Label1"h;```<br>```if $variable0 goto "Label1"h;``` |
```

Comment:
Add a hint to the description on a new line:
Differences between 30 and 32 are unclear.